src/features/devices/soil_humidity/soil_moisture_sensor.py

```diff
@@ -0,0 +1,57 @@
+from features.devices.device import Device, DeviceConfig, Topic
+from features.logger.logger import Logger
+from features.mqtt.mqtt import BaseMqttClient
+from utime import ticks_ms, ticks_diff, time
+from ujson import dumps, loads
+from features.analog_accessor.analog_accessor import AnalogAccessor
+import machine
+from drivers.sms import SMSDriver
+import uasyncio
+
+
+class Moisture(Topic):
+    pass
+
+
+class SoilMoistureSensor(Device):
```

Comment:
Tutaj fajnie było by jeszcze wykrywać wzrost z 20% np do 80% - mniej wiecej tak to wychodzi przy podlaniu i mozna by to zapamiętać i wrzucić w dodatkowy topic - ale to mozna dorobic jako nastepny etap